doc/source/release-notes/Percona-Server-5.6.48-88.0.rst

```diff
@@ -0,0 +1,32 @@
+.. _PS-5.6.48-88.0:
+
+================================================================================
+*Percona Server for MySQL* 5.6.48-88.0
+================================================================================
+
+:Date: May 18, 2020
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/5.6/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.6.48-88.0
+includes all the features and bug fixes available in
+`MySQL 5.6.48 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-48.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+Improvements
+================================================================================
+
+* :jirabug:`PS-5391`: RedHat-based distributions may set service configurations in a file saved in the ‘/etc/sysconfig' directory. The init script did not read any file in that directory. We have added the ability for the init.d script to read files in the ‘etc/sysconfig’ directory.
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-6953`: If the LOAD DATA process found a line with an escape character followed by UTF8 characters, then the process failed to load the line and any additional lines. This issue was a regression. (Upstream :mysqlbug:`98089`)
+* :jirabug:`PS-6945`: The tokubackup process exported an incorrect API. This API mismatch prevented large file backups.
+* :jirabug:`PS-6946`: The address and thread sanitizers found memory use after free and other issues in the tokubackup software..
+* :jirabug:`PS-7020`: In Ubuntu 20.04, only python3-mysqldb is available. We converted the MTR tests from python2 to python3.
```

Comment:
We converted python2 scripts to support both python2 and python3. It requires python2.6 or newer.

Reply:
Fixed the "to" and added the sentence about Python2.6.